app/models/mixins/supports_feature_mixin.rb

```diff
@@ -116,6 +116,33 @@ def supports?(feature)
       public_send("supports_#{feature}?")
     end
 
+    # all subclasses that are considered for supporting features
+    def permitted_subclasses
```

Comment:
We just have to be careful to not override the one defined in EMS

Reply:
we need the concept of all subclasses that are considered for supporting features. In Ems this is called `permitted_subclasses`. This is the method used by the ems version of this code that has been extracted to here.

Are you thinking we want a separate concept, or just concerned that we would override this method for EMS (instead of ensuring that EMS is overriding the method here)?

Reply:
I thought if we renamed the method in `ExtManagementSystem` to `permitted_subclasses` that would free up the `supported_subclasses` name to be used here.  In `ExtManagementSystem` we can alias one to the other.

Reply:
I don't see the need for this distinction but not my hill. done